app/routes/project/conversations/first.js

```diff
@@ -0,0 +1,11 @@
+import { get } from '@ember/object';
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  redirect() {
+    let { conversations } = this.modelFor('project.conversations');
+    if (get(conversations, 'length') > 0) {
+      this.transitionTo('project.conversations.conversation', get(conversations, 'firstObject'));
+    }
```

Comment:
I don't have a better way to handle this. Could use thoughts @begedin. The downside here is not getting the `.active` class on the project menu at all, which we want. We could force it some other way, but it's not ideal.

Reply:
Fixed by using `conversations.index` `redirect`.